src/crypto/generic-ops.h

```diff
@@ -60,14 +60,18 @@ namespace crypto { \
 namespace crypto { \
   static_assert(sizeof(std::size_t) <= sizeof(type), "Size of " #type " must be at least that of size_t"); \
   inline std::size_t hash_value(const type &_v) { \
-    return reinterpret_cast<const std::size_t &>(_v); \
+    std::size_t h; \
+    memcpy(&h, &_v, sizeof(h)); \
```

Comment:
This should arguably be updated to `std::addressof(_v)`. I still approved this PR anyway, as it is unlikely to be a problem because we rarely overload `operator&`.

Reply:
Done